<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tgt_page</class>
 <widget class="QWidget" name="tgt_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="tgttype_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Type of target</string>
   </property>
  </widget>
  <widget class="QComboBox" name="tgttype_comboBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="radial_dist_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Radial Distance</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rad_dist_lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="azimuth_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Azimuth</string>
   </property>
  </widget>
  <widget class="QLabel" name="speed_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Speed</string>
   </property>
  </widget>
  <widget class="QLabel" name="radial_mov_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Radial Movement</string>
   </property>
  </widget>
  <widget class="QLabel" name="time_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AZ_lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="speed_lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="rad_mov_comboBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="save_pushButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>330</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_pushButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>330</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="time_lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="timeoffset_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Time Offset</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="timeoffset_lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="vehicle_man_comboBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="playinloop_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Play in loop</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="loop_radioButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>89</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
